test/JDBC/input/babel-5179-vu-prepare.sql

```diff
@@ -0,0 +1,45 @@
+create view babel_5179_v1 as select cast(cast('12:45:37.123' as varchar) as time)
```

Comment:
please use meaningful testfile name instead of task number.

Reply:
changed name of file to cast-varchar-to-time-vu-*